optuna_dashboard/_app.py

```diff
@@ -309,6 +310,26 @@ def post_preference(study_id: int) -> dict[str, Any]:
         response.status = 204
         return {}
 
+    @app.put("/api/studies/<study_id:int>/preference_feedback_component_type")
+    @json_api_view
+    def put_component(study_id: int) -> dict[str, Any]:
```

Comment:
Could you update the view function name to elaborate the role of this api endpoint?